### PR TITLE
Scala 2.13 roadmap

### PR DESCRIPTION
**Pull request by adriaanm**
For publication on Monday Mar 6?

See https://github.com/scala/scala-dev/issues/324

### CODE REVIEW

File: news/_posts/2017-03-06-roadmap-2.13.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+Scala 2.13 is a library release, which means the language itself won't change. However, we continue to invest heavily in the implementation, with a single goal: the make the compiler faster. The first phase of this work involves benchmarking infrastructure, to guide our progress towards a faster compiler. After that, we will attack slowness on all fronts with all available tools. In addition to YourKit and other JVM profilers, which we have been using intensively over the years, we now have a JMH benchmarking harness for the compiler. We are also investigating lower-level performance (cpu caches, memory access patterns, JIT profiles, etc). We are eager to hear your ideas at the compiler performance theme issue. Please head over to https://github.com/scala/compiler-benchmark for the benchmarks and the issue tracker, or [discuss this theme here](https://github.com/scala/scala-dev/issues/322).
```
Comment (SethTisue) [inline news/_posts/2017-03-06-roadmap-2.13.md]: > which means the language itself won't change

perhaps the wording there is a bit too definite? maybe “won't see big changes”

File: news/_posts/2017-03-06-roadmap-2.13.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Finally, there are [many more things](https://github.com/scala/scala-dev/issues?q=is%3Aopen+is%3Aissue+milestone%3A2.13+label%3Awishlist) we'd like to include in 2.13. Let us know if you'd like to work on one of these. We're always happy to help!
```
Comment (SethTisue) [inline news/_posts/2017-03-06-roadmap-2.13.md]: this has to be weighed vs brevity, but maybe add "You'll find our active and friendly open-source community on [Scala Contributors](http://contributors.scala-lang.org), [scala/contributors](https://gitter.im/scala/contributors), and on GitHub issues and pull requests". the Discourse thing is still quite new and hasn't sunk in everywhere, and even the rise of Gitter isn't totally old news

File: news/_posts/2017-03-06-roadmap-2.13.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+In tandem, the modularisation effort of Scala 2.11 will continue as part of the [Scala Platform process](https://www.scala-lang.org/blog/2016/11/28/spp.html) to foster innovation in our eco-system while maintaining a stable core. The core consists of the collections and the other standard types, such as `Option`, `TupleN`, `Either` and `Try`.  Ultimately, we'd like provide even stronger binary compatibility guarantees for the core, with a vibrant complement of modules that evolve more quickly (offering only backwards compatibility).  [Discuss this theme]( https://github.com/scala/scala-dev/issues/323).
```
Comment (SethTisue) [inline news/_posts/2017-03-06-roadmap-2.13.md]: "the modularisation of the standard library (begun with Scala 2.11)"

**Comment by adriaanm** [comment]
Any thoughts/concerns? Would like to share this on scala-lang.

File: news/_posts/2017-03-06-roadmap-2.13.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+In tandem, the modularisation of the standard library (begun with Scala 2.11) will continue as part of the [Scala Platform process](https://www.scala-lang.org/blog/2016/11/28/spp.html), to foster innovation in our eco-system while maintaining a stable core. The core consists of the collections and the other standard types, such as `Option`, `TupleN`, `Either` and `Try`.  Ultimately, we'd like provide even stronger binary compatibility guarantees for the core, with a vibrant complement of modules that evolve more quickly (offering only backwards compatibility).  [Discuss this theme]( https://github.com/scala/scala-dev/issues/323).
```
Comment (jvican) [inline news/_posts/2017-03-06-roadmap-2.13.md]: Great to see the binary compatibility bits mentioned here.

**Comment by SethTisue** [comment]
couple tiny fixes in 06e6df7129a1ad4c4b9db37dd0598e7c111ab880

**Comment by heathermiller** [comment]
Hey guys, sorry that I missed this PR a few days back. This LGTM, thanks @adriaanm for writing it up!

**Comment by adriaanm** [comment]
No probs, thanks!

**Pull request merged**